resources/pithosctl.yaml

```diff
@@ -38,6 +38,10 @@ spec:
         product: pithos
     spec:
       serviceAccountName: pithos
+      securityContext:
```

Comment:
I also would suggest set `runAsUser: -1` which gravity replaces with `service-uid`. And delete group/fsgroup. The explicit setting of a specific user is needed only for one container in cassandra, where we need access to hostPath volume.

Reply:
I tried that, but I'm not sure it'll work. Let's see how it goes.